eng/Packages.Data.props

```diff
@@ -93,7 +93,7 @@
     <PackageReference Update="Azure.Messaging.ServiceBus" Version="7.10.0" />
     <PackageReference Update="Azure.Messaging.WebPubSub" Version="1.0.0" />
     <PackageReference Update="Azure.Monitor.Query" Version="1.1.0" />
-    <PackageReference Update="Azure.Identity" Version="1.6.0" />
+    <PackageReference Update="Azure.Identity" Version="1.7.0" />
```

Comment:
Note, I am bumping the central dependency to the version containing the breaking change @schaabs

Reply:
I had a PR out to do this, https://github.com/Azure/azure-sdk-for-net/pull/31284, but it seems the PR failed because the restore failed on one leg. I restarted the leg so it might merge in that PR.